index.html

```diff
@@ -1121,6 +1126,92 @@ <h2>
           <li><a>MiniAppWidgetResource.name</a></li>
       </ul>
     </section>
+    <section>
+      <h2>
+          <span its-locale-filter-list="en" lang="en"><dfn>I18n Solution</dfn></span>
+          <span its-locale-filter-list="zh-hans" lang="zh-hans"><dfn>I18n方案</dfn></span>
+      </h2>	  
+      <p its-locale-filter-list="en" lang="en">
+          To support the multiple languages, attributes such as name, short_name, and description can be set as a reference to a string resource. Typically, string resources are stored in the corresponding multi-language JSON file in the MiniApp package's i18n directory. The multi-language file’s name complies with IETF BCP 47.</p>
+      <p its-locale-filter-list="zh-hans" lang="zh-hans">
+          为了支持多语言能力name, short_name, description等参数可以被设置为字符串资源的索引。通常情况下字符串资源被保存在MiniApp包i18n目录下相应的多语言文件中。多语言文件命名规范符合IETF BCP 47。
+      </p>
+	  
+	  <p its-locale-filter-list="en" lang="en"> 
+	   The following example describes two simple multi-language files:
+	  </p>
+	  <p its-locale-filter-list="zh-hans" lang="zh-hans">
+	  以下例子描述2个简单的多语言文：
+	  </p>
+	  
+	  <p>i18n/en-US.json</p>
+	  </div> <pre aria-busy="false"><code class="hljs json">{
```

Comment:
You can just use `<pre class="example json">`. There is no need to add the hljs classes.

Reply:
Updated, thanks!